src/pybind/mgr/cephadm/templates/services/prometheus/prometheus.yml.j2

```diff
@@ -5,18 +5,19 @@ global:
 rule_files:
   - /etc/prometheus/alerting/*
 
-{% if alertmanager_sd_url %}
-alerting:
-  alertmanagers:
+
+{% macro monitoring_stack(basic_auth=false, url) -%}
```

Comment:
Since every preexisting block was checking for `secure_monitoring_stack` consider naming the new macro `monitoring_stack_secure` or something like that? Given that the whole block is disabled if `secure_monitoring_stack` is false...

Reply:
Or `configure_secure_monitoring_stack`.